ts/Extensions/Annotations/MockPoint.ts

```diff
@@ -6,12 +6,14 @@
 
 'use strict';
 
+import type Annotation from '../Annotations/Annotations';
+import type AnnotationChart from '../Annotations/AnnotationChart';
 import type MockPointOptions from './MockPointOptions';
 import type PositionObject from '../../Core/Renderer/PositionObject';
 import Series from '../../Core/Series/Series.js';
 
 declare module './MockPointOptions' {
-    interface MockPointOptions {
+    interface MockPointOptions { // @todo -> MockPointOptions
```

Comment:
Not part of Annotation class, I know, but could you move it in the d.ts file, please? We already mock with MockPoint, so time is right for this.